package.json

```diff
@@ -20,17 +19,14 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@centerforopenscience/ember-osf": "https://github.com/CenterForOpenScience/ember-osf.git#develop#2017-09-29T20:14:40Z",
+    "@centerforopenscience/ember-osf": "https://github.com/CenterForOpenScience/ember-osf.git#develop",
```

Comment:
idk if this was invalidationg the cache, but it seems to be working now?

Reply:
weird since a lot of people at COS use it without issue, but okay